### ISSUE TITLE
Can't run tests(.dll used by another process)

### ISSUE THREAD

**Issue by Haaxor1689** [formulahendry/vscode-dotnet-test-explorer#53]
I'm running into a breaking bug that prevents tests from running after few runs with code lens. The test explorer output print error: `Could not copy "{solution_folder}/project/bin/Debug/netstandard2.0/project.dll" to "bin/Debug/netcoreapp2.0/project.dll". Beginning retry {x} in 1000ms. The process cannot access the file '{solution_folder}/tests/bin/Debug/netcoreapp2.0/project.dll' because it is being used by another process.`

I am running this on .net core solution with `classlib` named `project` and `mstest` tests called `tests`. I found [this thread](https://stackoverflow.com/questions/16732195/vs2012-test-explorer-locks-native-dll-making-rebuilds-fail) that seems somehow related to this issue, in that it looks the compiled .dll is still used by some background debug process.

**Comment by stefanforsberg** [comment]
By "few runs with code lens", do you mean clicking on the code lens run / debug test options?

![image](https://user-images.githubusercontent.com/358570/38565776-75b5301e-3ce2-11e8-918c-9ced6da36e1c.png)

**Comment by Haaxor1689** [comment]
Yes it happens when I run code via the lens options. Also I've noticed it is much more probable to bug, the longer I have the debugger paused on a breakpoint or maybe when I inspect variables of current scope. I wasn't able to closer specify the cause so far.

**Comment by stefanforsberg** [comment]
Those are added by the c# extension (C# for Visual Studio Code (powered by OmniSharp)). If you disable .net core text explorer for your repo you should be able to reproduce the issue. If it only occurs when you have .net core text explorer enabled then please report back.
